_rules/table-header-cell-has-assigned-cells-d0f69e.md

```diff
@@ -27,16 +27,16 @@ acknowledgments:
 
 ## Applicability
 
-This rule applies to any HTML element with the [semantic role][] of [rowheader][] or [columnheader][] for which all of the following is true:
+This rule applies to any HTML element with is a [semantic][semantic role] [rowheader][] or [columnheader][] for which all of the following is true:
```

Comment:
I feel this requires a precision in the def of "Semantic Role" that we can say "an element is a semantic foo" to mean "has a semantic role of foo".

Reply:
Solved in https://github.com/act-rules/act-rules.github.io/pull/1671/files#diff-a0b8349df87c854df34f5481b6c6a983ca594f8b4ea62347239579f80d6d4e0fR16